tests/integration/vlan_test.py

```diff
@@ -64,3 +77,50 @@ def vlan_interface(ifname, vlan_id):
                 ]
             }
         )
+
+
+@contextmanager
+def two_vlans_on_same_interface():
```

Comment:
Since is is about two vlans on eth1, it makes more sense to call it `two_vlans_on_eth1`.